keyboards/viktus/minne/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2023 Viktus Design LLC
```

Comment:
This file should be removed

(it only contains `config_common.h` which should no longer be included)